addons/api/addon-test-support/helpers/indexed-db.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This test helper can help set up and cleanup the database in your tests.
+ *
+ * ```js
+ * import { setupIndexedDb } from 'api/test-support/helpers/indexed-db';
+ *
+ * module('Acceptance | my test', function(hooks) {
+ *  setupApplicationTest(hooks);
+ *  setupIndexedDb(hooks);
+ *
+ *   // add your actual tests here
+ * });
+ * ```
+ *
+ * @param hooks
+ */
+export function setupIndexedDb(hooks) {
+  hooks.beforeEach(function () {
+    const indexedDb = this.owner.lookup('service:indexed-db');
+    indexedDb.setup(`test-indexed-db-${Date.now()}`);
+  });
+
+  hooks.afterEach(async function () {
+    const indexedDb = this.owner.lookup('service:indexed-db');
```

Comment:
nonblocking: any reason for not defining indexedDb outside the `beforeeach` scope and use that in `aftereach` instead of looking it up again?

Reply:
Can I do that? I thought that because these are test helpers they have to be self contained within the hooks?

Reply:
Seems like we [can](https://github.com/hashicorp/boundary-ui/blob/3f731867ac36d284ea92e413caad862b691768f3/addons/auth/tests/unit/session-stores/cookie-test.js#L12), we do this in a lot of places

Reply:
But this is an `addon-test-support` helper, it doesn't have that context right?